ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.8.yaml

```diff
@@ -0,0 +1,793 @@
+  optional: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: assisted-service
+    env:
+      RELEASE_TAG_PREFIX: ocm-2.8
+    test:
+    - ref: assisted-baremetal-images-publish
+- as: operator-publish
+  postsubmit: true
+  steps:
+    test:
+    - ref: assisted-baremetal-operator-publish
+- as: assisted-operator-catalog-publish
+  postsubmit: true
+  steps:
+    env:
+      OPERATOR_MANIFESTS_TAG_TO_PIN: ocm-2.8
+      REGISTRY_BUNDLE_REPOSITORY_TAG: ocm-2.8
+      REGISTRY_CATALOG_REPOSITORY_TAG: ocm-2.8
+    test:
+    - ref: assisted-baremetal-operator-catalog-publish
+- as: assisted-operator-catalog-publish-verify
+  run_if_changed: ^(deploy/olm-catalog/.*)$
+  steps:
+    env:
+      REGISTRY_BUNDLE_REPOSITORY_NAME: assisted-service-operator-bundle-test
+      REGISTRY_CATALOG_REPOSITORY_NAME: assisted-service-index-test
+    test:
+    - ref: assisted-baremetal-operator-catalog-publish
+- as: lint
+  commands: |
+    export GOCACHE=/tmp/
+    export GOLANGCI_LINT_CACHE=/tmp/.cache
+    export GOPROXY=https://proxy.golang.org
+    make lint
+  container:
+    from: assisted-service-lint
+  skip_if_only_changed: ^dashboards/|^openshift/|^docs/|^sites/|\.md$|^(LICENSE|OWNERS|OWNERS_ALIASES)$
+- as: verify-generated-code
+  skip_if_only_changed: ^dashboards/|^openshift/|^docs/|^sites/|\.md$|^(LICENSE|OWNERS|OWNERS_ALIASES)$
+  steps:
+    test:
+    - ref: assisted-verify-generated-code
+- as: unit-test
+  commands: |
+    export REPORTS=/tmp/reports
+    export CODECOV_TOKEN=$(cat /tmp/secret/codecov-token)
+    export GINKGO_REPORTFILE=${ARTIFACT_DIR}/junit_unit_test.xml
+    make ci-unit-test
+  container:
+    from: assisted-service-unit-test
+  secret:
+    mount_path: /tmp/secret
+    name: assisted-service-codecov-token
+  skip_if_only_changed: ^dashboards/|^openshift/|^docs/|^sites/|\.md$|^(LICENSE|OWNERS|OWNERS_ALIASES)$
+- as: unit-test-postsubmit
+  commands: |
+    export REPORTS=/tmp/reports
+    export CODECOV_TOKEN=$(cat /tmp/secret/codecov-token)
+    export GINKGO_REPORTFILE=${ARTIFACT_DIR}/junit_unit_test.xml
+    make ci-unit-test
+  container:
+    from: assisted-service-unit-test
+  postsubmit: true
+  secret:
+    mount_path: /tmp/secret
+    name: assisted-service-codecov-token
+- as: subsystem-aws
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
+  run_if_changed: ^(cmd/.*|deploy/.*|hack/.*|internal/.*|pkg/.*|tools/.*|subsystem/.*|Dockerfile\..*|Makefile|go\.mod|go\.sum|swagger.yaml)$
+  steps:
+    allow_best_effort_post_steps: true
+    test:
+    - as: subsystem-deploy-and-run
+      cli: latest
+      commands: |
+        export BUILD_TYPE=CI
+        export REPORTS=${ARTIFACT_DIR}
+        export TARGET=oc
+        export PERSISTENT_STORAGE="False"
+        export GENERATE_CRD='false'
+        unset GOFLAGS
+        make ci-deploy-for-subsystem
+        oc get pods
+        make test
+      dependencies:
```

Comment:
sorry I'm only now seeing this
we don't set image-service 2.8 in this job?

Reply:
What do you mean? the assisted-service specified in the dependency is built as a part of this file

Reply:
yes, but if not overridden it's using image-service of master, which could make some inconsistencies

Reply:
ah then we have and issue in all the other branches from what I see

Reply:
done!